Basiq-34:Remove 'Select Account' screen

albatahmet opened the pull request:
For Basiq-34 task, from account verification steps, steps 4 and 5 were disabled, after entering username and password for account, redirected to home page.

disabled pages;
AccountVerificationFormStep4SelectAccount.js
AccountVerificationFormStep5Summary.js

albatahmet committed:
Page 4 and page 5 disable

Page 4 and page 5 disable

k-cem-fd commented on components/AccountVerificationForm/AccountVerificationFormProvider.js:
```diff
@@ -306,10 +306,6 @@ function useBasiqConnection({ currentStep, userId }) {
       return;
     }
     if (completed) {
```
You can remove this if condition block completely, there is no need to leave it as empty @albatahmet

k-cem-fd commented on components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js:
```diff
@@ -10,18 +11,37 @@ export function AccountVerificationFormStep3LoadingSteps() {
   const [isResumeModalOpen, openResumeModal, closeResumeModal] = useTernaryState(false);
 
-  const { basiqConnection, goForward } = useAccountVerificationForm();
+  const { basiqConnection ,finish,updateAccountVerificationFormState} = useAccountVerificationForm();
```
Please use spaces correctly @albatahmet

k-cem-fd commented on components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js:
```diff
@@ -10,18 +11,37 @@ export function AccountVerificationFormStep3LoadingSteps() {
+  });
+
+  const errorOrNoData = error || !data || data.length === 0;
+  async function autoSelectedAccount () {
```
Why did you create a function to select an account automatically ? You can put this logic in fetchAccounts method and set currentAccount

albatahmet committed:
Basiq-34

Removed unnecessary lines of code.
Space usage errors fixed.
The automatic user selection function has been removed and placed in the service.

albatahmet committed:
Basiq-34

Fail Test fixed

MahmutKasimoglu commented on components/AccountVerificationForm/AccountVerificationFormStep3LoadingSteps.js:
```diff
@@ -58,8 +70,47 @@ export function AccountVerificationFormStep3LoadingSteps() {
+    axios
+      .get('/api/accounts', { params: { userId } })
+      .then(res => {
+        console.log(res.data);
```
seems like a forgotten console.log

albatahmet replied:
I forgot it, thank you very much

MahmutFD committed:
basiq-43: Updated the Account Used in Transaction Detail to use the current Account.

albatahmet committed:
Basiq-34

AccountVerificationFormStep3LoadingSteps.js
    console.log, comment line and unnecessary spaces removed

albatahmet committed:
Basiq-34

Updated for 2 conflicting PRs

MahmutFD committed:
Merge branch 'origin/features/albatahmet/BASIQ-34_Remove_Select_Account_Screen' of https://github.com/flyingdonkey-it/personal-finance-app-API-v3.0 into origin/features/mahmutk/basiq-43/Transaction_Static_Detail_text

MahmutFD committed:
BASIQ-43: Removed setting accountId to session storage on step 4

MahmutFD committed:
BASIQ-43/Removed Unneccesary space

MahmutKasimoglu committed:
Merge pull request #30 from flyingdonkey-it/origin/features/mahmutk/basiq-43/Transaction_Static_Detail_text

BASIQ-43/ Transaction Detail static text
